monai/transforms/signal/radial_fourier.py

```
@@ -0,0 +1,225 @@
+        self,
+        n_bins_list: Sequence[int] = (32, 64, 128),
+        return_types: Sequence[str] = ("magnitude",),
+        normalize: bool = True,
+    ) -> None:
+        super().__init__()
+        self.n_bins_list = n_bins_list
+        self.return_types = return_types
+        self.normalize = normalize
+
+        if not n_bins_list:
+            raise ValueError("n_bins_list must not be empty")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)